### ISSUE TITLE
staging: virtglrenderer broken

### ISSUE THREAD

**Issue by dtzWill** [NixOS/nixpkgs#43651]
## Issue description

Another cause of being unable to build NixOS VM's (#43650 ).

...What's going on here, folks? :(

### Steps to reproduce

```console
$ nix build -I testpkgs=https://github.com/NixOS/nixpkgs/archive/dca7b10fd52371548870f7a8aba90c39574ca89e.tar.gz  '(import <testpkgs> {}).virglrenderer'
```

Build log: https://gist.github.com/dtzWill/eb4bd6cb8738017b9ff8cf3e3fcaaa48

Also seen by ofborg [here](https://github.com/NixOS/nixpkgs/pull/43648#issuecomment-405439267).

## Technical details

See other issue.

**Comment by dtzWill** [comment]
maybe fixed with this: 2bd8e98eba5798ae948335da6db304fe10626904

So if a fix is made on staging-next, should it not be made on staging?
Is there a way to mark staging as being known completely-broken? (is this a state it will be more frequently now?)

(cc https://github.com/NixOS/rfcs/pull/26)

I don't build my system on staging or anything but I do send PR's there often and it's usually in good shape -- certainly at least in terms of packages building.

I'd pick that myself (assuming it fixes at least this bit?) but I'm not sure if that's how this goes.

As staging-next gets fixes, should it just be continuously merged into staging (or at least "as needed")?

**Comment by vcunat** [comment]
I had tested the commit to fix the virgl build and that's how I found about the problem.  I haven't yet worked out good details of the new workflow :-)  Merging sounds OK, so let me do it at least now.
